3.2/s2i/bin/usage

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+man -l /help.1
```

Comment:
`groff-base` should also be installed in the `Dockerfile`, so we make sure it is available.